testsuite/features/core/proxy_branch_network.feature

```diff
@@ -205,5 +205,5 @@ Feature: Setup Uyuni for Retail branch network
 @proxy
 @private_net
   Scenario: The terminals should not reach the server
-    Then "sle_client" should not communicate with the server
-    And "sle_minion" should not communicate with the server
+    Then "sle_client" should not communicate with the server using "eth1"
+    And "sle_minion" should not communicate with the server using "eth1"
```

Comment:
it should also not communicate using `eth0`...

Also, this is fragile. If you remember, we considered several times simplifying the network setup by having only one card on client and minion. That would fail with this new syntax.

Reply:
Well, so then we have a problem with our product, as it communicates successfully through eth0.
Could you please take care of that network issue?